src/core/layout/gel-defaults.js

```diff
@@ -204,7 +204,6 @@ export const config = screen => {
             id: "skip",
             channel: buttonsChannel(screen),
             action: ({ screen }) => {
-                gmi.sendStatsEvent("skip", "click");
```

Comment:
will we not lose all skip stats with removing this?

Reply:
Nope, it's not used anywhere else.

Reply:
yeah that's what I mean, I thought we were getting 2 skip stats being sent and that we do actually want a skip stat.